core/client/app/components/gh-user-invited.js

```diff
@@ -16,8 +17,10 @@ export default Ember.Component.extend({
     actions: {
         resend: function () {
             var user = this.get('user'),
-                notifications = this.get('notifications');
+                notifications = this.get('notifications'),
+                self = this;
```

Comment:
I've already got a local commit that updates the majority of the exports on the ember side - I wanted something repetitive to do whilst I was mulling over a completely different issue! I can get that finished and pushed if we're heading in that direction.